### ISSUE TITLE
Repository Statistics Page

### ISSUE THREAD

**Issue by colehudson** [WSULib/ouroboros#46]
Per our earlier conversation, we need a repository stats page. It could be useful to show things like size of repository, size of certain datastreams, etc. Showing these things, particularly datastream size, will be easily done when we have adjusted our Solr core to include datastream size and mimetype with each object doc.

**Comment by ghukill** [comment]
Agreed!  In the process of updating solr indexing, will introduce datastream info as new dynamic field under the pattern, `ds_*`.

A couple come to mind:
  * `ds_[DSID]_size_i`
  * `ds_[DSID]_mime`
